history/cultures/dragonmaw.txt

```diff
@@ -15,11 +15,15 @@
 	discover_innovation = innovation_motte 
 	#
 	discover_innovation = innovation_city_planning
+	#
+	discover_innovation = innovation_dragon
 }
 508.1.1 = {
 	discover_innovation = innovation_catapult 
 	#
 	discover_innovation = innovation_currency_01
+	#
+	discover_innovation = innovation_wyvern
```

Comment:
So this would be limited only to Durotaric? and maybe maknothal?

Reply:
> So this would be limited only to Durotaric? and maybe maknothal?

Limited? It should be regional. Cultures that went to the Barrens like the Frostwolf, Durotaric, Warsong should have it.